.github/workflows/psalm.yml

```diff
@@ -1,34 +0,0 @@
-name: Psalm
```

Comment:
I would prefer to have Psalm be a separate CI badge.

Reply:
Got it, brought the workflow file back. Just out of curiosity, why would you prefer that? Should I also add PHPStan (which I've added in #195) as an extra file/badge too?

Reply:
With two badges: If CI is failing, if it's only a Psalm issue, it's easier to tell at a glance when looking at the README file months down the line.

PHPStan generally doesn't cause the flakiness that Psalm does when they update their rules.

Reply:
Makes sense, thanks.